editoast/src/core/pathfinding.rs

```diff
@@ -105,7 +105,7 @@ pub enum PathfindingCoreResult {
 }
 
 /// A successful pathfinding result. This is also used for STDCM response.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, ToSchema)]
+#[derive(Default, Serialize, Deserialize, Clone, Debug, PartialEq, ToSchema)]
```

Comment:
nit: can we use `cfg_attr(test` to better indicate that it's just a shortcut we use in tests?

Reply:
Changed in 6ec2e277.